lib/server.js

```diff
@@ -102,9 +102,9 @@ function parsePOST(req) {
 
 function logResponse(opts, response, startTime) {
     const latency = Date.now() - startTime;
-    let logLevel = 'trace/request';
+    let logLevel = 'info/request';
```

Comment:
Exactly. Which is why I think we should separate it: access logs: only path, query, resp code and latency with info level, and debug/trace level for what is currently being output. The less data we have per info log entry, the less the burden on the disk.